mbed_tools_lib/logging.py

```diff
@@ -0,0 +1,34 @@
+"""Helpers for logging errors according to severity of the exception."""
+import logging
+from typing import Any
+
+LOGGING_FORMAT = "%(levelname)s: %(message)s"
+
+
+def log_exception(logger: Any, exception: Exception) -> None:
```

Comment:
Should `logger` be `Any`? 🤔 I'd ask somebody experienced with typing.

Reply:
I guess that we need the logger to have the methods `error` and `debug` that expects those args, so it's probably better to restrict to anything that extends `logging.Logger`.